app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -356,7 +356,7 @@ In order to be paid the IRP, you must have a UK bank account in your name (this
 
 If you are a teacher or a salaried teacher trainee, you must also have a [UK National Insurance number](https://www.gov.uk/apply-national-insurance-number). You can apply for this once you arrive in England, as long as your visa gives you the right to work and you have a confirmed job.  
 
-Teachers and salaried trainees should receive a single payment from the Department for Education shortly after the end of their first term of employment as a teacher. Fee-paying trainees will receive a single payment from their provider around the end of their first term of training. You will be paid as and when your eligibility requirements have been checked – we cannot guarantee an exact payment date.  
+Teachers and salaried trainees should receive a single payment from the Department for Education shortly after the end of the term of employment as a teacher in which they apply. Fee-paying trainees will receive a single payment from their provider around the end of the term of training in which they apply. You will be paid as and when your eligibility requirements have been checked – we cannot guarantee an exact payment date.  
```

Comment:
@laurahermionetennant92 (DfE) is this the 1st mention on the page? Can it just say DfE instead of in full?

Reply:
@laurahermionetennant92 You'll

Reply:
I know we would normally abbreviate DfE after first use but it is such a long page and English is not their first language I worry they'll forget what it stands for. This is also the reason I keep on. using IRP in full because in user research they forgot what it stood for.